### PR TITLE
ROC-2230: Add defendant timeline

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[ROC-2230](https://tools.hmcts.net/jira/browse/ROC-2230)

### Change description ###
This PR adds defendant time line for full defence.

### CCD Spreadsheet changes ###
https://github.com/hmcts/cmc-integration-tests/pull/166

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 1]
```diff
@@ -277,6 +281,29 @@
...
+        </td>
+        <td>
+          {{ event.description | trim }}
+        </td>
+      </tr>
+      {% endfor %}
+      {% endif %}
+
+      <tr class="no-border">
+        <td><strong>Comments about claimant's timeline:</strong></td>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: should be an apostrophe not a quote here

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 2]
```diff
@@ -277,6 +281,29 @@
           {{ responseDefence }}
         </td>
       </tr>
+      {% if claim.events is defined %}
+      <tr class="no-border">
+        <td colspan="2" style="white-space: pre-line"><strong>Defendant's timeline of what happened:</strong></td>
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: same here

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 3]
```diff
@@ -277,6 +281,29 @@
...
+          {{ event.description | trim }}
+        </td>
+      </tr>
+      {% endfor %}
+      {% endif %}
+
+      <tr class="no-border">
+        <td><strong>Comments about claimant's timeline:</strong></td>
+        <td>
+          {{ timelineComment }}
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: check what should go here if not provided

Comment (sabahirfan) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: done

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html (hunk 2 above)
Comment (sabahirfan) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: done

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html (hunk 1 above)
Comment (sabahirfan) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: done

**Comment by sabahirfan** [comment]
`dev-build` is passing and that matter atm.

https://dev-build.reform.hmcts.net/job/hmcts/job/cmc-claim-store/view/change-requests/job/PR-266/

**Comment by sabahirfan** [comment]
Closing this as https://github.com/hmcts/cmc-claim-store/pull/274 is raised for the same.

**Pull request closed without merging**